### PR TITLE
Add changes requested by TR (noref)

### PR DESCRIPTION
**Pull request by dmpop**
A few tweaks based on TR's feedback.

### CODE REVIEW

File: xml/poc_crowbar_guide.xml [hunk 1]
```diff
@@ -77,16 +77,7 @@
...
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     <emphasis>Networking Plug-Ins</emphasis> &cloud; natively supports open
-     source SDNs via Open vSwitch, harnessing the power of DPDK in &cloudos;.
-     For more flexibility, &cloud; provides support for third-party tools from
-     Cisco, Midokura, Infoblox, Nuage Networks, PLUMgrid and even VLAN bridging
-     solutions.
+     plug-ins available from SUSE Enterprise Storage and other storage vendors like EMC, NetApp.
```
Comment (asettle) [inline xml/poc_crowbar_guide.xml]:
```suggestion
plug-ins are available from SUSE Enterprise Storage and other storage vendors like EMC, NetApp.
```

Comment (dmpop) [reply xml/poc_crowbar_guide.xml]: My bad. Done!

Comment (csymons-suse) [reply xml/poc_crowbar_guide.xml]: use &ses; entity

Comment (dmpop) [reply xml/poc_crowbar_guide.xml]: Done.

**Pull request merged**